scrapper.py

```diff
@@ -44,18 +78,58 @@ def get_search_urls(self):
         pass
 
 
+class HTMLParser:
+    def __init__(self, article_url, article_id):
+        self.article_url = article_url
+        self.article_id = article_id
+        self.article = Article(self.article_url, self.article_id)
+
+    def parse(self):
+        response = requests.get(self.article_url, HEADERS)
+        article_bs = BeautifulSoup(response.text, 'html.parser')
+        self._fill_article_with_text(article_bs)
+        return self.article
+
+    def _fill_article_with_text(self, article_bs):
+        pass
+
+
 def prepare_environment(base_path):
     """
     Creates ASSETS_PATH folder if not created and removes existing folder
     """
-    pass
+    try:
+        os.rmdir(base_path)
+    except FileNotFoundError:
+        os.mkdir(base_path)
 
 
 def validate_config(crawler_path):
     """
     Validates given config
     """
-    pass
+    with open(crawler_path) as file:
+        config = json.load(file)
+
+    seed_urls = config['seed_urls']
+    total_articles = config['total_articles_to_find_and_parse']
+
+    if not isinstance(total_articles, int) or total_articles <= 0:
+        raise IncorrectNumberOfArticlesError
+
+    if total_articles > 100:
```

Comment:
I recommend taking a bigger number as 100 is a required number of articles to find and parse, so it is a bit odd that this code is expected to work for 100 items but is going to complain about 101